### ISSUE TITLE
XML Schema Validation in test?

### ISSUE THREAD

**Issue by lsetiawan** [ODM2/WOFpy#208]
Because of issue #202 with `unitCode` that @emiliom had found, it got me thinking to maybe do an automatic XML validation with the schema file as a test? This would help ensuring that we are following the schema.

**Comment by emiliom** [comment]
Yeah, something along those lines would be good to implement in tests. But not this month!
